src/chip/README.md

```diff
@@ -0,0 +1,28 @@
+# @dojo/widgets/chip widget
+
+Dojo's `Chip` widget allows content and particularly user inputs to be displayed more clearly and enables interaction such as removing entered values or clicking them to trigger some action
+
+
+## Features
+
+- Displays a label, optional icon, and an optional close icon
+
+## Example Usage
+
+```typescript jsx
+// Basic chip
+<Chip label="Displayed value" iconRenderer={() => <Icon type="plusIcon"/>}/>
+```
+
+## Theming
```

Comment:
We no longer need to put theming information in here as it's picked up by the example app